client/my-sites/navigation/index.jsx

```diff
@@ -19,17 +22,31 @@ class MySitesNavigation extends React.Component {
 	};
 
 	render() {
+		let sidebar = null;
+		if ( 'jetpack-cloud' === this.props.application ) {
```

Comment:
Seems like this logic could be placed directly when rendering, no need to store it in a var. Or, since everything is the same, the logic could just determine the component to render, since the props are the same.

Reply:
:+1: